cheat.php

```diff
@@ -71,15 +71,16 @@
 $ZonePaces = [];
 $OldScore = 0;
 $LastKnownPlanet = 0;
-
-Msg( "{background-blue}Welcome to SalienCheat for SteamDB" );
+$BestPlanetAndZone = 0;
 
 if( ini_get( 'precision' ) < 18 )
 {
 	Msg( '{teal}Fixed php float precision (was ' . ini_get( 'precision' ) . ')' );
 	ini_set( 'precision', '18' );
 }
 
+Msg( "{background-blue}Welcome to SalienCheat for SteamDB" );
```

Comment:
Can move this back where it was

Reply:
yes, sorry for the loooonq delay, I tried to amend my commit but it doesn't work that way.